build/js/live-editor.output_pjs.js

```diff
@@ -1602,10 +1614,38 @@ window.PJSOutput = Backbone.View.extend({
             // but PImage may be mutable, so that might not work.
             this.imageCache[file] = img;
         }.bind(this));
-    },
+        
+        _.each(sounds, function(file) {
```

Comment:
Totally agree.  Are there other places in KA that have a "loading resources..." progress bar?